src/metabase/public_settings/premium_features.clj

```diff
@@ -473,9 +470,9 @@
 
   ###### `:feature`
 
-  A keyword representing a premium feature which must be present for the EE implementation to be used. Use `:any` to
-  require a valid premium token with at least one feature, but no specific feature. Use `:none` to always run the
-  EE implementation if available, regardless of token.
+  A keyword representing a premium feature which must be present for the EE implementation to be used. Use `:none` to
+  always run the EE implementation if available, regardless of token (WARNING: this is not recommended for most use
+  cases. You probably want to gate your code by a specific premium feature.)
```

Comment:
Keeping `:none` as an option (with a warning) since there's at least one place (enterprise model loading) where we genuinely want the code to run for EE instances regardless of token. But that's more of an infra thing; it shouldn't be used for feature implementations.

Reply:
You're referring to the `resolve-enterprise-model` function right?

wdyt if we move that logic into a normal function in `metabase.models` and call it with a guard `(when config/ee-available?)`

this way we can get rid of the `:none` flag all together

Reply:
Good idea

Reply:
Actually, I still don't think we can get rid of `:none`. Audit v2 will use it for `ensure-audit-db-installed!` because it should run on every EE build at startup. (So that if a token is later added, the audit DB is already available). And there might be other things like this in the future.